code/game/machinery/kitchen/microwave.dm

```diff
@@ -32,6 +32,14 @@
 												/obj/item/weapon/reagent_containers/food/condiment,
 												/obj/item/weapon/reagent_containers/dropper)
 
+	var/rig_meal = 1
```

Comment:
shouldn't this default to 0? looks like every first cook on a microwave will fail

Reply:
You're right, thats a mistake.